src/librustdoc/clean/mod.rs

```diff
@@ -1819,17 +1825,26 @@ pub(crate) fn clean_ty<'tcx>(ty: &hir::Ty<'tcx>, cx: &mut DocContext<'tcx>) -> T
         TyKind::Array(ty, ref length) => {
             let length = match length {
                 hir::ArrayLen::Infer(..) => "_".to_string(),
-                hir::ArrayLen::Body(anon_const) => {
+                hir::ArrayLen::Body(const_arg) => {
                     // NOTE(min_const_generics): We can't use `const_eval_poly` for constants
                     // as we currently do not supply the parent generics to anonymous constants
                     // but do allow `ConstKind::Param`.
```

Comment:
Oh this is nice cleanup I guess. we shouldn't have anon consts containing generic parameters under this PR anymore so this is no long an issue. Probably best to clean this up in a future PR though since this is already quite large :3

Reply:
I think this is just a note for future work, not something to be resolved now—right?

Reply:
Yep